crates/oxc_linter/src/snapshots/no_await_in_loop.snap

```diff
@@ -0,0 +1,87 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_await_in_loop
+---
+  ⚠ eslint(no-await-in-loop): Unexpected `await` inside a loop.
+   ╭─[no_await_in_loop.tsx:1:38]
+ 1 │ async function foo() { while (baz) { await bar; } }
```

Comment:
AwaitExpression may have a long argument, Shall we just point to the keyword `await`?